.github/workflows/create-branch-from-linear.yml

```
@@ -0,0 +1,77 @@
+      labels:
+        description: "Comma-separated labels (e.g., bug, documentation)"
+        required: false
+        type: string
+
+jobs:
+  create_branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set inputs for manual run or real comment
+        id: set_vars
+        run: |
```

actionlint (1.7.7)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)